src/sensors/hx711s.py

```diff
@@ -1,3 +1,59 @@
 from HX711Multi import HX711_Multi
 
-#TODO: create in X1-AV-38
\ No newline at end of file
+'''
+Constants
+'''
+
+# Taring settings
+NUMBER_OF_SAMPLES = 200
+MAX_AWAY_FROM_STD = 0.3
+
+# Init settings
+CLK_PIN = 27
+GAIN = 128
+DEBUGGER_MODE = False
+
+DATA_PIN_1 = 17
+DATA_PINS = [DATA_PIN_1]
```

Comment:
Add comment or add them all

Reply:
There should be 3 CLK_PINs because having all 12 HX711s on one clock can apparently cause issues tho I haven't been able to test it.